java/meeting-scheduling/src/main/resources/META-INF/resources/app.js

```diff
@@ -212,7 +212,7 @@ function renderScheduleByPerson(schedule) {
             meet.preferredAttendances.forEach(attendance => {
                 if (meet.requiredAttendances.map(a => a.person).indexOf(attendance.person) === -1) {
                     const byPersonElement = $("<div />").append($("<div class='d-flex justify-content-center' />").append($(`<h5 class="card-title mb-1"/>`).text(meet.topic)));
-                    byPersonElement.append($("<div class='d-flex justify-content-center' />").append($(`<span class="badge text-bg-info m-1" style="background-color: ${pickColor(meet.id)}" />`).text("Preferred")));
+                    byPersonElement.append($("<div class='d-flex justify-content-center' />").append($(`<span class="badge text-bg-info m-1" style="background-color: ${color.bg};color: ${color.fg}" />`).text("Preferred")));
```

Comment:
Is there any reason for why the meetings are not colored as in the other examples?